.github/workflows/rust-pipeline.yml

```diff
@@ -0,0 +1,101 @@
+name: Rust pipeline
```

Comment:
Might be overly pedantic but I think pipeline isn't a term used in github-world ^_^